### DISCUSSION TITLE
only build tea

### DISCUSSION THREAD

**Discussion by harfiansol** [teaxyz/white-paper#276] [General]
im personaly only bullish whit tea

**Comment by Ufik0203** [comment]
keep support on TEA

**Comment by mixaeljr** [comment]
A decentralized protocol for developer

**Comment by engr0001** [comment]
looking forward to what tea will offer. its a great project......

**Comment by maniaca** [comment]
Support project tea

**Comment by Brox0077** [comment]
TEA project on support
